content/applications/marketing/sms_marketing/essentials/sms_essentials.rst

```diff
@@ -176,6 +175,31 @@ is visualized as different graphs and charts, which can be sorted and grouped in
    :align: center
    :alt: Reporting page in SMS Marketing.
 
+.. tip::
+   SMS messages can be sent using server actions in Odoo. To do so, in :ref:`developer mode
+   <developer-mode>`, navigate to :menuselection:`Settings app --> Technical menu --> Automation -->
+   Automated Actions`. Click :guilabel:`Create` or :guilabel:`New` to create a new action.
+
+   Enter an :guilabel:`Action Name` and select a :guilabel:`Model` to implement this action on. Set
```

Comment:
```suggestion
   Enter an :guilabel:`Action Name`, and select a :guilabel:`Model` to implement this action on. Set
```
do you think it's important to reference/explain the 'Active' toggle switch that appears between 'Model' and 'Trigger'?